src/pages/scripts/getting-started/index.md

```diff
@@ -69,6 +69,7 @@ To edit a script shown in the Scripts panel, select the script and choose 'Edit
 Let's add the following, to update the font and size of the text
 
 ```js
+let { app } = require("indesign");
```

Comment:
Not required since we are building on top of the existing script in this tutorial.